API.md

```diff
@@ -74,9 +74,9 @@ Filename extensions of user-editable documents.
 ### `history`
 History of edits done to `doc`.
 
-### `editCount`
+### `docChangeCounter`
```

Comment:
```suggestion
### `docVersion`
```

Reply:
I'd prefer if this property were called `docVersion` rather than `docChangeCounter`, so that we could in principle implement this using e.g. a `UUID` given as a `string` or some hash of the file contents in the future. This gives more flexibility to the implementer of the API without making the property any less useful to the plugin author.

What do you think?

Reply:
Would you expect the `docVersion` to change back on undo? And are you fine if the version is indicated by a number - basically as the counter does?

Reply:
I see a potential disadvantage: Performance. When we want to reliably have a hash of the current state of the document, it might take a bit. 

Another question also: how would a counter or version behave when we support a multi file edition?

Reply:
I guess `docVersion` is a little more implementation agnostic and frees us up to change later if we so choose.

Reply:
Ok I am on board.